lib/src/validator_builder.dart

```diff
@@ -27,7 +27,7 @@ class ValidationBuilder {
   }
 
   static final RegExp _emailRegExp = RegExp(
-      r"^[a-zA-Z0-9.a-zA-Z0-9.!#$%&'*+-/=?^_`{|}~]+@[a-zA-Z0-9\-\_]+(\.[a-zA-Z]+)*$");
```

Comment:
I know what you did is a reasonable validation for most of the end users, but it's technically a breaking change and opinionated. I believe we should leave previous regexp-s as is, because the email protocol [spec](https://www.w3.org/Protocols/rfc822/) does not specify any length related limitations.

I know again this probably won't be an issue for 99% of the users. However personally I would prefer sticking to standards rather than relying on my own opinions. For another example the regex does not require email domain to be 2nd level domain name, one could use "a@b" as a domain name and it would pass the test, because it's indeed a valid email address according to the standards.

Reply:
Thank you for taking the time to review the changes I've made. I understand your opinion and that makes sense to me. I am open to undoing it ASAP, but how would you feel about adding an option so that normal, expected usage with these limits are supported easily.

For example;

`ValidationBuilder().email().withLengthLimit().build();`

Reply:
Hey @SemihGursoy ,

What do you think about this option:

```dart
// optional constructor argument
ValidationBuilder(emailRegexp: RegExp('custom regexp'))

// or global configuration
ValidationBuilder.globalOptions.emailRegexp = RegExp('...')
```

Reply:
Hey, that would be great for my use case. Especially defining it as a global, since this validation is needed on multiple screens. I will start working on it.

Reply:
If you are alright with that option, I would appreciate if you can revert that regexp change and limit scope of this PR to the translation change only. Then I'll merge this PR and I can quickly add customizable regexp option and push a new version.

Reply:
@themisir I am very happy with that option, thank you! I just reverted my previous commit.